### PR TITLE
Create betaActivity.jsonld

### PR DESCRIPTION
**Pull request by aeidi89**
openMetadataInitiative/openMINDS_instances#100

### CODE REVIEW

File: instances/measuredSignalType/betaActivity.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "knowledgeSpaceLink": null,
+  "name": "beta activity",
+  "preferredOntologyIdentifier": "https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=D001611",
+  "synonym": [
+    "Beta rhythm",
+    "Beta Rhythms",
+    "Rhythm, Beta",
+    "Rhythms, Beta",
+    "Beta frequency",
+    "Beta wave"
```
Comment (lzehl) [inline instances/measuredSignalType/betaActivity.jsonld:19]: comma is missing (thats why the checks fail). please correct

File: instances/measuredSignalType/betaActivity.jsonld [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "@id": "https://openminds.ebrains.eu/instances/measuredSignalType/betaActivity",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/measuredSignalType",
+  "definition": "Beta activity is an electromagnetic oscillation in the frequency range of 8–12 Hz arising from synchronous and coherent (in phase / constructive) electrical activity. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Beta_wave)]",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0101243",
+  "knowledgeSpaceLink": null,
+  "name": "beta activity",
+  "preferredOntologyIdentifier": "https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=D001611",
+  "synonym": [
```
Comment (lzehl) [inline instances/measuredSignalType/betaActivity.jsonld:13]: please remove the capitalizations

File: instances/measuredSignalType/betaActivity.jsonld [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/measuredSignalType/betaActivity",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/measuredSignalType",
+  "definition": "Beta activity is an electromagnetic oscillation in the frequency range of 8–12 Hz arising from synchronous and coherent (in phase / constructive) electrical activity. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Beta_wave)]",
```
Comment (lzehl) [inline instances/measuredSignalType/betaActivity.jsonld:7]: "Beta activity is a neural oscillation in the mid frequency range (typically between 12-30 Hz) arising from synchronous and coherent electrical activity in the brain. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Beta_wave)]"

@apdavison would you agree with this definition?

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by UlrikeS91** [comment]
Replaced by https://github.com/openMetadataInitiative/openMINDS_instances/pull/84

**Pull request closed without merging**